cg/server/api.py

```diff
@@ -145,73 +145,71 @@ def submit_order(order_type):
 @BLUEPRINT.route("/cases")
 def cases():
     """Fetch cases."""
-    records = db.cases(days=31)
-    count = len(records)
-    return jsonify(cases=records, total=count)
+    cases = db.cases(days=31)
+    count = len(cases)
+    return jsonify(cases=cases, total=count)
 
 
 @BLUEPRINT.route("/families")
 def families():
     """Return families."""
     if request.args.get("status") == "analysis":
-        records = db.cases_to_mip_analyze()
-        count = len(records)
+        cases = db.cases_to_mip_analyze()
```

Comment:
`cases_to_mip_analyze` does not exist 🤔

Reply:
Created a PR for it https://github.com/Clinical-Genomics/cg/pull/1825